src/components/CropSelector/CropCalendarView/CropCalendarView.js

```diff
@@ -596,7 +408,13 @@ const CropCalendarViewComponent = (props) => {
             <TableBody className="calendarTableBodyWrapper">
               {activeCropData.length > 0 ? (
                 <>
-                  <RenderCrops active={true} cropData={activeCropData} />
+                  <RenderCrops
+                    active={true}
+                    cropData={activeCropData}
+                    setModalOpen={setModalOpen}
+                    modalOpen={modalOpen}
+                    setModalData={setModalData}
+                  />
                   <RenderCrops active={false} cropData={activeCropData} />
                 </>
               ) : (
```

Comment:
line 421, 392, 391, 372 and possibly a few more have empty ternary.  
i also see on 341, 321, 301, 281, seem to have very similar blocks of code.  we can use a function call here to return the same thing with new parameters.  
i bet the operator on 382 can be rewritten in one ternary